site/en/blog/third-party-scripts/index.md

```diff
@@ -0,0 +1,264 @@
+---
+title: 'How can browsers optimize the user-experience of third-party scripts?'
+description: >
+  Driving changes for better loading of third-party resources on the web
+layout:
+  'layouts/blog-post.njk'
+date: 2022-08-07
+authors:
+  - addyosmani
+hero: 'image/1L2RBhCLSnXjCnSlevaDjy3vba73/kng4NSnIkUQ9BU4RLT3p.jpeg'
+alt: >
+  ALT TEXT
+tags:
+  - performance
+  - news
+---
+
+[Third-party](https://almanac.httparchive.org/en/2021/third-parties) (3P) scripts and embeds are heavily used across the web today. They provide out-of-the-box solutions for embedding social media, videos, analytics, live chat, advertising, A/B testing, personalization, and others. Third-party embeds are a necessary part of modern websites that allow site owners to focus on their core competencies, while offloading standard but critical functions to external providers.
+
+When both first party and third parties on a web page work in harmony, we have a very performant outcome and a great user experience. However, this requires significant effort from both engineering and business teams and is often overlooked, resulting in less than performant web pages and a [negative impact](https://web.dev/third-party-javascript/#performance) to user-centric metrics such as the [Core Web Vitals](https://web.dev/vitals/). This is detrimental to both parties and creates missed opportunities in businesses. Could we do better here?
+
+We have a future vision of the web where third-party scripts and resources provide the intended business value with minimal regression to the performance or user experience of websites that use them in the browser. This would allow users to experience, ideally, faster page loads.
```

Comment:
```suggestion
We have a future vision of the web where third-party scripts and resources provide the intended business value with minimal regression to the performance or user experience of websites that use them in the browser. This would allow users to ideally experience faster page loads.
```

Reply:
Minor tweak for flow here.